doc/man3/SSL_get_error.pod

```diff
@@ -85,7 +85,9 @@ protocol level.
 It is safe to call SSL_read() or SSL_read_ex() when more data is available
 even when the call that set this error was an SSL_write() or SSL_write_ex().
 However, if the call was an SSL_write() or SSL_write_ex(), it should be called
-again to continue sending the application data.
+again to continue sending the application data.if you get B<SSL_ERROR_WANT_WRITE>
+from SSL_write() or SSL_write_ex() then you should not do any other B<IO> operation
```

Comment:
`then you should not do any other operation that could trigger IO other than to repeat the previous SSL_write() call.`

Reply:
ok, Modified